src/Pages/Buzzer/index.jsx

```diff
@@ -0,0 +1,7 @@
+export const BuzzerComponent = () => {
```

Comment:
This should be importing `useParams` like the Display/index.jsx page so that you can add the `key` from the path.